index.d.ts

```diff
@@ -64,12 +64,12 @@ type AjaxRequestCallback = (response: string, x: XMLHttpRequest) => void;
 
 // type LoadCallback = (error: any, result: string | false) => void;
 
-export default class I18NextXhrBackend implements i18next.BackendModule<BackendOptions> {
+export default class I18NextXhrBackend implements BackendModule<BackendOptions> {
   constructor(services?: any, options?: BackendOptions);
   init(services?: any, options?: BackendOptions): void;
-  readMulti(languages: string[], namespaces: string[], callback: i18next.ReadCallback): void;
-  read(language: string, namespace: string, callback: i18next.ReadCallback): void;
-  loadUrl(url: string, callback: i18next.ReadCallback): void;
+  readMulti(languages: string[], namespaces: string[], callback: ReadCallback): void;
```

Comment:
I wonder if this should actually use `MultiReadCallback` from i18next... but it seems like that's not currently being used internally either (https://github.com/i18next/i18next/blob/master/index.d.ts#L706).